syntaxes/yml.tmLanguage.YAML-tmLanguage

```diff
@@ -0,0 +1,220 @@
+$schema: 'https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json'
+name: 'Yseop Markup Language'
+scopeName: source.yseop
+patterns:
+    -
```

Comment:
Wut? Isn’t there a more concise way to write this inclusion list? I don’t even get what that means, in terms of structures (lists, etc.). Is that a list of singleton maps? Gosh. Sure you can’t provide a list as value for `include`? :open_mouth:

Reply:
I used a tool to convert the JSON file into this one first. 
It seems that 
```-
    include: '#foo'
```
has the same meaning as
```
- {include: '#foo'}
```

Reply:
I almost tried to write an AWK script to change that stuff but screw that (mostly because of indentation and stuff).
:woman_shrugging: